### PR TITLE
COMMERCE-12430 Organisation Management Chart to Support Search 

### PR DESCRIPTION
**Pull request by pietrobocale**
https://liferay.atlassian.net/browse/COMMERCE-12430

Base PR:
https://github.com/liferay-commerce/liferay-portal/pull/4292

### CODE REVIEW

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrganizationManagementWidget.testcase [hunk 1]
```diff
@@ -252,4 +303,787 @@ definition {
...
+
+			task ("And highlight 'Org 2' with the black color") {
+				AssertElementPresent(
+					key_entry = "Org 2",
+					locator1 = "CommerceOrganizationManagementWidget#CHART_DISCOVERED_ENTRY");
+			}
+		}
+
+	}
+
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrganizationManagementWidget.testcase]: -------------

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrganizationManagementWidget.testcase [hunk 2]
```diff
@@ -252,4 +303,787 @@ definition {
...
+			JSONOrganization.addOrganization(
+				organizationName = "SubOrg 1",
+				parentOrganizationName = "Org 1");
+
+			for (var accountsList : list "Account A,Account B_1,Account B_2") { 		
+				JSONAccountEntry.addAccountEntry(
+					accountEntryName = "${accountsList}",
+					accountEntryType = "Business");
+
+				if(${accountsList} != "Account B_1") {
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrganizationManagementWidget.testcase]: I need to add the user admin admin to Account B_1

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrganizationManagementWidget.path [hunk 3]
```diff
@@ -55,6 +55,61 @@
...
+	<td>//div[contains(@class,'org-chart-container')]//*[name()='g' and contains (@class,"discovered")][${key_row}]//*[name()='text' and contains (.,'${key_entry}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CHART_ENTRY</td>
+	<td>//div[contains(@class,'org-chart-container')]//*[name()='g' and contains (@class,"chart-item")]//*[name()='text' and contains (.,'${key_entry}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CHART_PLUS</td>
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrganizationManagementWidget.path:94]: Remove this path

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrganizationManagementWidget.path [hunk 4]
```diff
@@ -55,6 +55,61 @@
...
+	<td>//div[contains(@class,'org-chart-container')]//*[name()='g' and contains (@class,"open-actions-wrapper")]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CHART_SEARCH_RESULTS</td>
+	<td>//div[contains(@class,'org-chart-result-helper') and contains (.,'${key_searchResults}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CHART_ADD_ORGANIZATION</td>
```
Comment (pietrobocale) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrganizationManagementWidget.path:104]: Remove this path

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:sf

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
ci:stop

**Comment by pietrobocale** [comment]
ci:test:commerce-functional

**Comment by pietrobocale** [comment]
ci:test:relevant

**Comment by pietrobocale** [comment]
Relevant failures not related to the PR

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**